Apps/W1/Email - Outlook REST API/app/src/EmailOutlookAPISetup.Table.al

```diff
@@ -29,6 +29,12 @@ table 4509 "Email - Outlook API Setup"
             Caption = 'Redirect URL';
             DataClassification = CustomerContent;
         }
+        field(5; TenantID; Text[36])
```

Comment:
Maybe type should be `Guid` (to be clear what data will be sored)?

Reply:
I've set it to Text to allow the standard value "Common". Another solution could be to get the tenant ID, but OnPremise this would not work properly

Reply:
Maybe "Common" can be represented as a null guid in the table, and then we could have a small helper function that substitutes the tenant ID into the string with the logic to convert null guid into `Common`?

Reply:
I've done this, could you please give me feedback. Is it in the way you thought?